src/object.d

```diff
@@ -3167,7 +3167,7 @@ struct Test
 size_t hashOf(T)(auto ref T arg, size_t seed = 0)
 {
     import core.internal.hash;
-    return core.internal.hash.hashOf(arg, seed);
```

Comment:
Note that this code has a bug in it - the `seed` is interpreted as the length of `arg` !!